install.sh

```diff
@@ -151,7 +151,7 @@ function curl () {
 }
 
 # This will create the directories.
-crew_folders="bin cache doc docbook include lib/crew/packages lib$LIB_SUFFIX libexec man sbin share var etc/crew/meta etc/env.d tmp/crew/dest"
+crew_folders="lib64 bin cache doc docbook include lib/crew/packages lib$LIB_SUFFIX libexec man sbin share var etc/crew/meta etc/env.d tmp/crew/dest"
```

Comment:
`$LIB_SUFFIX` abstracts the library folder which is not `lib64` on arm and i686 architecture.  Is `$LIB_SUFFIX` not set to `64` on your system?

Reply:
done!

Reply:
With `lib$LIB_SUFFIX` we don't need `lib64`...